src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/TrackingOptInModal.java

```diff
@@ -146,6 +146,11 @@ public void isTrackingRequestsNotSend(List<String> elementsList,
 
   public void isTrackingRequestSend(List<String> elementsList,
                                     NetworkTrafficInterceptor networkTrafficInterceptor) {
+    try {
+      Thread.sleep(25000);
+    } catch (InterruptedException e) {
+      e.printStackTrace();
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Use a logger to log this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1148)